Samples/LibVLCSharp.WinForms.Sample/Form1.Designer.cs

```diff
@@ -0,0 +1,68 @@
+﻿using LibVLCSharp.Shared;
+using System.Windows.Forms;
+
+namespace LibVLCSharp.WinForms.Sample
+{
+    partial class Form1
+    {
+        /// <summary>
+        /// Required designer variable.
+        /// </summary>
+        private System.ComponentModel.IContainer components = null;
+
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Windows Form Designer generated code
+
+        LibVLC _libVLC;
```

Comment:
_libVLC and _mp will need to be declared in ` Form1.cs` because this file needs to be untouched by hand.